libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx

```diff
@@ -7,17 +7,20 @@ import { GenericFormContext } from '../../providers';
 import { useContext } from 'react';
 
 /* eslint-disable-next-line */
-export interface GenericFormSubmitButtonProps {}
+export interface GenericFormSubmitButtonProps {
+  name: string;
+}
 
 const StyledGenericFormSubmitButton = styled.div``;
```

Comment:
What's the use of it if there is no styling ?

Reply:
sorry you mean my change or empty styled.div?

Reply:
empty styled.div

Reply:
this empty-styled. div is used to get wrapped with another content. and this structure is followed on other files/folders too. not sure if this is a good idea to remove it.